moveit_core/collision_detection_bullet/include/moveit/collision_detection_bullet/bullet_integration/contact_checker_common.h

```diff
@@ -102,15 +102,26 @@ inline collision_detection::Contact* processResult(ContactTestData& cdata, colli
       cdata.res.distance = contact.depth;
     }
   }
+
   ROS_DEBUG_STREAM_NAMED("collision_detection.bullet", "Contact btw " << key.first << " and " << key.second
                                                                       << " dist: " << contact.depth);
+  // case if pair hasn't a contact yet
   if (!found)
   {
-    cdata.res.collision = true;
+    if (contact.depth < 0)
```

Comment:
`<=`? Though it probably doesn't matter.

Reply:
done